### ISSUE TITLE
Need to decide on default vcus value for GPUs

### ISSUE THREAD

**Issue by sestephens73** [ut-parla/Parla.py#28]
The default value for `vcus` is currently 1 for GPUs. Previously it was set to `attrs["MultiProcessorCount"]`, which ought to be equivalent to the number of SMs on the GPU.

The value is set [here](https://github.com/ut-parla/Parla.py/blob/9e70d29742b4a7f19641fe870e0c6c4c4649e361/parla/cuda.py#L88) in Parla.py/parla/cupy.py.

We need to have a discussion on which would be preferable. 1 is simpler, but SM count would be more expressive and potentially useful for nodes with multiple different types of GPUs.

**Comment by arthurp** [comment]
I don't think this is a "default". I would say it's the definition of VCU for specific architectures. 

The default number of VCUs a task uses is always 0 currently. And probably should be 1.

This fact of needing a default of all tasks may actually provide a useful insight into how VCUs should be defined for each device. Maybe a VCU should be the amount of compute used by a "typical" computational kernel on that device. This would be very hard to define in it's own right though. So this will need thought.
